components/ProjectModal/index.tsx

```diff
@@ -1,8 +1,9 @@
 'use client';
 
 import React, { useEffect, useState } from 'react';
-import Modal from 'react-modal';
+import { FiX, FiZap } from 'react-icons/fi';
 import { queryProjectbyId } from '../../api/supabase/queries/query';
+import * as texts from '../../styles/texts';
 import { Project } from '../../types/schema';
 import * as styles from './styles';
```

Comment:
If possible, I think we should import all of the styled components individually as their names instead of as styles so that we don't have to start everything with "styles.---". It looks kinda hard to read with all of the "styles.---".

Reply:
yeah I think in my previous project we also imported all styling individually instead of using styles. or texts. So I agree that we should change it to import individually. E.g. in texts.ts I imported all the fonts as: `import { CoinbaseMono, CoinbaseSans, CoinbaseText } from './fonts';` instead of `import * as fonts from ./fonts`

I will also make this announcement on slack